types/node/index.d.ts

```diff
@@ -671,6 +671,24 @@ declare namespace NodeJS {
         ref(): void;
         unref(): void;
     }
+
+    class Module implements NodeModule {
+        static runMain(): void;
+        static wrap(code: string): string;
+
+        static Module: typeof Module;
+
+        exports: any;
+        require: NodeRequireFunction;
+        id: string;
+        filename: string;
+        loaded: boolean;
+        parent: NodeModule | null;
```

Comment:
Can you also change this and below from `NodeModule` to `Module`?

I believe the end goal will be to replace all usages of `NodeModule` with `Module`.

Reply:
done. I remove also the `implements NodeModule` as it caused issues with other modules which augment `NodeModule` and as you indicate the target is anyhow to get rid of it.